src/Exec.hs

```diff
@@ -152,22 +153,13 @@ runStep properties (ShellCmd workdir cmd mprop haltOnFailure) = do
   unless (null outmsg) $ putOut @s outmsg -- show step normal output, if any
   unless (null errmsg) $ putErr @s errmsg -- show step error output, if any
   let properties' = case mprop of Nothing -> properties
-                                  Just prop -> Map.insert prop (normalize outmsg) properties
+                                  Just prop -> Map.insert prop (trim outmsg) properties
```

Comment:
This is changing the behavior. We don't want to trim the property's value at the start nor do we want to eliminate trailing spaces. I'll restore the previous behavior which was doing the minimal possible change.

Reply:
Ok, I'll change it back

Reply:
Done